articles/200_migration_guide_from_ros1.md

```diff
@@ -121,6 +133,25 @@ Apply the following changes to use `ament_cmake` instead of `catkin`:
 * Replace ROS 1 dependencies like `roscpp` with ROS 2 dependencies like `rclcpp`.
   Using `rclcpp` will also require to link explicitly against one rmw implementation, commonly the default is provided by the [rmw_implementation](https://github.com/ros2/rmw_implementation/tree/master/rmw_implementation) package.
 
+#### Unit tests
+
+If you are using gtest
+
+* replace `CATKIN_ENABLE_TESTING` with `AMENT_ENABLE_TESTING`
+* replace `catkin_add_gtest` with `ament_add_gtest`
+* add a `<test_depend>ament_cmake_gtest</test_depend>`
+*
+
+It is recommended to turn on the automatic linter unittests by adding these lines just below `if(AMENT_ENABLE_TESTING)`:
```

Comment:
I think this needs the extra comment that it is only recommended if the package follows the ROS 2 conventions.